webxrlayers-1.bs

```diff
@@ -553,9 +555,10 @@ When this method is invoked, it MUST run the following steps:
          <dd> [=reject=] |promise| with an {{InvalidStateError}} and return it.
          <dd> 
          </dl>
+    1. let |depthsupport| be true if |context| is a {{WebGL2RenderingContext}} or the {{WEBGL_depth_texture}} extension is enabled in |context|.
```

Comment:
Ooh! This is a good check, thanks for considering it!

I think that down below if the user requests depth and `depthsupport` is `false` we should actually reject since we can't adequately fulfill the request. UAs will probably want to accompany it with a console error explaining why, but that's not part is not really something that we can/should spec.